### ISSUE TITLE
ScanNet training of PTv2

### ISSUE THREAD

**Issue by antao97** [Pointcept/Pointcept#63]
Hi, I follow your running command using 4 3090 cards, but the performance is not good. Can you tell me whether the training process is normal or not?
![截屏2023-07-30 上午11 13 54](https://github.com/Pointcept/Pointcept/assets/41320807/7028ab1f-14e1-44ab-9f1e-11805a751cdc)

**Comment by Gofinge** [comment]
The training curve is not normal, could you provide a detailed training log?

**Comment by antao97** [comment]
[train-lovasz.log](https://github.com/Pointcept/Pointcept/files/12208686/train-lovasz.log)
[train-base.log](https://github.com/Pointcept/Pointcept/files/12208687/train-base.log)
Here are the training log of lovasz and base.

The lr and training loss:
![截屏2023-07-30 下午3 28 29](https://github.com/Pointcept/Pointcept/assets/41320807/c6025b91-580f-491a-a1f2-9463b92c5c76)
![截屏2023-07-30 下午3 28 34](https://github.com/Pointcept/Pointcept/assets/41320807/0cb4353a-32be-4078-bb1e-20a57a1b8605)
The black curve is base, and the blue curve is lovasz.

**Comment by antao97** [comment]
> Hi, I follow your running command using 4 3090 cards, but the performance is not good. Can you tell me whether the training process is normal or not? ![截屏2023-07-30 上午11 13 54](https://user-images.githubusercontent.com/41320807/257030001-7028ab1f-14e1-44ab-9f1e-11805a751cdc.png)

Same as lr and training loss, the black curve is base and the blue curve is lovasz.

**Comment by Gofinge** [comment]
Hi, I checked the config and ran the config in my local environment. Unfortunately, I can not reproduce the issue you mentioned.  The original training curve is available in the old PTv2 codebase readme (https://tensorboard.dev/experiment/2eyHRBfrRyyywUqej9yh2w/#scalars&_smoothingWeight=0) and train log and test log is also available. I suggest making sure your local code, environment, and data are correct, then try to reproduce our results.

**Comment by antao97** [comment]
Thanks for your detailed reply! 

I first checked whether my spconv installation is correct. I trained models on the ScanNet and the SemanticKITTI datasets and get mIoU 75.85% and 70.51%. So my dataset processing has no problem.

I remember that I ran python setup.py install failed the first time. I followed your instructions to install cuda 11.3 version pytorch, but my server is installed with cuda 11.7. So I created a new environment and install cuda 11.7 version pytorch. Then, I ran python setup.py install successfully. But before this command running, I ignored to remove all dependencies under libs/pointops produced when I install pointcept at the first time. Therefore, my code ran strangely in PTv2. 

I removed the dependencies under libs/pointops and reinstalled pointcept. Then I found pointcept run correctly. I successfully reproduce the reported results using your pretrained model on the SemanticKITTI dataset (#42). I get val mIoU 70.86% and you report val mIoU 70.3% in README.

Now I'm running training on the ScanNet dataset and the process is normal. Many thanks for your prompt reply!
